redirects-list.json

```
@@ -73,6 +73,14 @@
   "^/doc/user-guide/dvc-internals(/.*)?$                                                  /doc/user-guide/project-structure/internal-files$1",
   "^/doc/user-guide/dvcignore$                                                            /doc/user-guide/project-structure/dvcignore-files",
   "^/doc/user-guide/contributing(/.*)?$                                                   /doc/contributing$1",
+  "^/doc/user-guide/large-dataset-optimization$                                           /doc/user-guide/data-management/large-dataset-optimization",
+  "^/doc/user-guide/external-dependencies$                                                /doc/user-guide/data-management/importing-external-data",
+  "^/doc/user-guide/pipelines/external-dependencies$                                      /doc/user-guide/data-management/importing-external-data",
+  "^/doc/user-guide/managing-external-data$                                               /doc/user-guide/data-management/managing-external-data",
+  "^/doc/user-guide/pipelines/external-outputs$                                           /doc/user-guide/data-management/managing-external-data",
+  "^/doc/user-guide/running-dvc-on-windows$                                               /doc/user-guide/how-to/running-dvc-on-windows",
+  "^/doc/user-guide/setup-google-drive-remote$                                            /doc/user-guide/how-to/setup-google-drive-remote",
+  "^/doc/user-guide/visualizing-plots$                                                    /doc/user-guide/experiment-management/visualizing-plots",
   "^/doc/understanding-dvc(/.*)?$                                                         /doc/user-guide/what-is-dvc",
```

Unrelated but we can update this now
```suggestion
  "^/doc/understanding-dvc(/.*)?$                                                         /doc/user-guide/overview",
```

---

Or maybe even remove it altogether (it's been there for a long time)

```suggestion
```